src/search.c

```diff
@@ -810,6 +814,7 @@ static enum swupd_code download_manifests(struct manifest **MoM)
 enum swupd_code search_main(int argc, char **argv)
 {
 	int ret = SWUPD_OK;
+	int search = SWUPD_NO;
```

Comment:
Why not using ret instead of creating a new variable?

Reply:
Done